### PR TITLE
Cause of Death System design document

### PR DESCRIPTION
**Pull request by enzonr**
THis is the System design document for the Cause of Death Package.

And it should essentially be in a similar hierarchical level as the system design doc for COVID.

### CODE REVIEW

File: commonmark/en/content/metadata/Cause_of_death.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
```
Comment (Philip-Larsen-Donnelly) [inline commonmark/en/content/metadata/Cause_of_death.md]: This is not a markdown file. It is HTML. 
It also seems to be in the wrong place. I presume it should be removed?

File: src/commonmark/en/content/metadata/Cause_of_death.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
```
Comment (Philip-Larsen-Donnelly) [inline src/commonmark/en/content/metadata/Cause_of_death.md]: This file seems to be in the correct place.
But it is not markdown, it is HTML.

Please convert to markdown.

**Review by Philip-Larsen-Donnelly — CHANGES_REQUESTED** [review]
Please remove the wrongly placed file and convert the other to markdown.

File: src/commonmark/en/content/metadata/Cause_of_death.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+# Cause of Death Workflow
+
+The cause of death module uses a framework based off of the ICD (International Classification of Diseases) method of recording cause of death data. The [international medical certificate of cause of death](http://apps.who.int/iris/handle/10665/40557) serves as the main source of information in this process. In order for the information to be entered in DHIS2, the medical certificate of cause of death must first be filled in (this will likely be a manual process and not filled in directly in DHIS2). This information will then be entered into DHIS2 using one of the program types that has been selected. The data entry form itself is a replica of the paper based medical certificate of cause of death. For more information/background on this certificate please refer to the ICD SMoL Training Manual.|
+
+![](https://lh5.googleusercontent.com/i-BpEnYyZF98nuDvfPjm6tp6L2BEG-LlFwFfo3jLzAkqQ3JlyoK1cJXYO133GB-f3O5sCOJTSar6FHCEtCIUZAnu3hNKxyZwDGOKs3_0rLL839MRAbNP6VomazjyLFROy_PNys_r)
```
Comment (Philip-Larsen-Donnelly) [inline src/commonmark/en/content/metadata/Cause_of_death.md:41]: These should be changed to local file references  (resources/images/...)

**Review by Philip-Larsen-Donnelly — CHANGES_REQUESTED** [review]
I have put your pros/cons and stage/descriptions into tables. You should check that they make sense.

**Comment by Philip-Larsen-Donnelly** [comment]
Note that the metadata stuff has also moved to a new repository since you forked this one :grimacing: 
@YuryRog may be able to help you to migrate it across.

**Comment by enzonr** [comment]
Oh no. Yury, can you help at some point?

I did not mean to do that!

**Pull request closed without merging**